dialogue-core/src/test/java/com/palantir/dialogue/core/PinUntilErrorChannelTest.java

```diff
@@ -87,12 +90,34 @@ public void channels_are_shuffled_initially_and_successful_requests_stay_on_chan
 
     @Test
     public void various_error_status_codes_cause_node_switch() {
-        testStatusCausesNodeSwitch(429);
         for (int errorStatus = 500; errorStatus < 600; errorStatus++) {
             testStatusCausesNodeSwitch(errorStatus);
         }
     }
 
+    @ParameterizedTest
+    @MethodSource("qosCodes")
+    void qos_responses_do_not_cause_node_switch(int status) {
```

Comment:
308 is specifically meant to force us to switch, 503 as well, I think this should verify only 429

Reply:
good point - was a bit hasty!